features/built_in_matchers/include.feature

```diff
@@ -11,6 +12,9 @@ Feature: `include` matcher
 
     expect([1, 2]).to include(1)
     expect([1, 2]).to include(1, 2)
+    expect([1, 2]).to include(be_an(Integer))
```

Comment:
Mmh... It's different from the other ones.

```ruby
        it { is_expected.to include(be_odd.and be < 10) } # combined matcher
        it { is_expected.to include(be_odd) } # call just odd?
        it { is_expected.to include(be_an(Integer)) } # full matcher checking type
```

What a about an alias for include, like the one you propose on the other PR? `have_a_member`?